### PR TITLE
testing updates after inactivity

### PR DESCRIPTION
**Pull request by topepo**

### CODE REVIEW

**Comment by hfrick** [comment]
Whoo, it skips the right tests now! 

Status on Matrix: the `1.6-2` binaries for Windows are now on CRAN, so I'm going to lower the minimum version in the skips to that (rather than `1.6-3` as it is currently so that we don't skip these tests for (potentially) too long, waiting for the next release).

**Comment by hfrick** [comment]
Looks like we need the following versions for those three packages (based on an issue comment on the lme4 repo): 
Matrix: 1.6-2 
RcppEigen: 	0.3.3.9.4
lme4: 1.1-35.1 

The session info shows that we are getting those versions for [lme4](https://github.com/tidymodels/extratests/actions/runs/6828744639/job/18573499027#step:6:13105) and [RcppEigen](https://github.com/tidymodels/extratests/actions/runs/6828744639/job/18573499027#step:6:13190) but not yet for [Matrix](https://github.com/tidymodels/extratests/actions/runs/6828744639/job/18573499027#step:6:13113). That one gets installed in version 1.6-1.1 from CRAN.

So I'm also going to change the skip for `lme4` to the current CRAN version `1.1-35.1` and remove the Remote for it (it did not install the dev version based on that anyway).

File: tests/testthat/test-glmnet-poisson.R [hunk 1]
```diff
@@ -121,19 +121,25 @@ test_that("formula interface can deal with missing values", {
   expect_true(is.na(f_pred$.pred[1]))
 })
 
-test_that('error traps', {
```
Comment (simonpcouch) [inline tests/testthat/test-glmnet-poisson.R:124]: Appreciating how this was split up. :)

File: tests/testthat/_snaps/sparse-glmnet.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
     Code
       mold(rec, mlc_churn, blueprint = sparse_bp)
-    Error <rlang_error>
-      `data` must only contain numeric columns.
+    Condition
+      Error in `recompose()`:
```
Comment (simonpcouch) [inline tests/testthat/_snaps/sparse-glmnet.md:6]: Is this a regression? Not sure if this is a change in rlang or our own, but if this seems like a step backward I say we file an issue.

Comment (simonpcouch) [reply tests/testthat/_snaps/sparse-glmnet.md:6]: edit: referring to the reference to `recompose()` newly showing up.

File: tests/testthat/_snaps/glmnet-poisson.md [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
-
----
+# model errors on missing penalty value
 
     Code
       poisson_reg() %>% set_engine("glmnet") %>% fit(mpg ~ ., data = mtcars[-(1:4), ])
-    Error <rlang_error>
-      For the glmnet engine, `penalty` must be a single number (or a value of `tune()`).
+    Condition
+      Error in `.check_glmnet_penalty_fit()`:
```
Comment (simonpcouch) [inline tests/testthat/_snaps/glmnet-poisson.md:6]: Same thought re: new `call` output.

File: tests/testthat/test-parsnip-model-formula.R [hunk 4]
```diff
@@ -1,5 +1,5 @@
 test_that('error without model formula (workflow, no tune)', {
-  skip_if_not_installed("parsnip", min_version = "1.1.1.9001")
+  skip_if_not_installed("parsnip", minimum_version = "1.1.1.9001")
```
Comment (simonpcouch) [inline tests/testthat/test-parsnip-model-formula.R:2]: 🤪

**Review by simonpcouch — APPROVED** [review]
We are back on the horse!🐴

File: tests/testthat/test-parsnip-model-formula.R (hunk 4 above)
Comment (hfrick) [reply tests/testthat/test-parsnip-model-formula.R:2]: same! https://github.com/tidymodels/extratests/pull/108#discussion_r1383629276

**Comment by hfrick** [comment]
Re error call:
If it's coming from hardhat: that hasn't been touched since before extratests broke down. If it's coming from parsnip: that totally needs various updates on how error calls are handled. So I think we'd pass over this once we do parsnip errors - which is one of those "epic" issues. So maybe not necessary to open issues specifically for those two error calls? But happy to do so if you think that's the better approach!

**Comment by simonpcouch** [comment]
That makes sense, thanks! :)

**Comment by hfrick** [comment]
I shall merge then! 🎉

**Pull request merged**